commands/registry.go

```diff
@@ -144,6 +144,21 @@ func Repository() *Command {
 		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
 	)
 
+	listRepositoriesV2Desc := `This command retrieves information about repositories in a registry, including:
+  - The repository name
+  - The latest manifest of the repository
+  - The latest manifest's latest tag, if any
+  - The number of tags in the repository
+  - The number of manifests in the repository
+`
```

Comment:
I wonder if there is some additional info we could provide to users that would help them choose between `list` and `list-v2`?

Reply:
I'm not sure. I was thinking just the fact of `v2` and seeing the different results would be self explanatory. I'm open to suggestions

Reply:
Since we've gone with hiding v1, it probably not very important now.